server/routes/routerRequestHandlers/removeFormDataFromSession.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request, Response } from 'express'
```

Comment:
This looks like a new file, but is actually a rename of the previous `removeInductionFormsFromSession` (see next file)